### ISSUE TITLE
NaN beta value seen, may be due to an SNC (a constant SNP)

### ISSUE THREAD

**Issue by ChrisGottsacker** [Parks-Laboratory/fastlmm_pipeline#1]
stdout
```
NaN beta value seen, may be due to an SNC (a constant SNP)
NaN beta value seen, may be due to an SNC (a constant SNP)
... (exactly same message repeated 2621 times total)
NaN beta value seen, may be due to an SNC (a constant SNP)
```

stderr
```
std=.0.000000 has illegal value for SNPs[:][3]
std=.0.000000 has illegal value for SNPs[:][4]
std=.0.000000 has illegal value for SNPs[:][4]
std=.0.000000 has illegal value for SNPs[:][1]
std=.0.000000 has illegal value for SNPs[:][2]
... (variant of same message repeated 51725 times total)
std=.0.000000 has illegal value for SNPs[:][4]
std=.0.000000 has illegal value for SNPs[:][6]

Traceback (most recent call last):
  File "fastlmm_node.py", line 119, in <module>
    run_fastlmmc(dataset, output_dir, pheno_index, covFile, species, maxthreads, featsel=featsel, 
	exclude=exclude, condition=condition)
  File "fastlmm_node.py", line 63, in run_fastlmmc
    df = single_snp(test_snps = test_snps, pheno = pheno_file, K0 = mat_snps)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    association/single_snp.py", line 207, in single_snp
    runner = runner)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    util/mapreduce.py", line 180, in map_reduce
    result = runner.run(dist)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    util/runner/Local.py", line 27, in run
    result = run_all_in_memory(distributable)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    util/runner/__init__.py", line 41, in run_all_in_memory
    return work.reduce(result_sequence)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    util/mapreduce.py", line 81, in reduce
    return self.reducer(output_seq)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/fastlmm/
    association/single_snp.py", line 189, in reducer_closure
    frame = pd.concat(frame_sequence)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/pandas/
    tools/merge.py", line 845, in concat
    copy=copy)
  File "/var/lib/condor/execute/slot1/dir_2281862/python/lib/python2.7/site-packages/pandas/
    tools/merge.py", line 878, in __init__
    raise ValueError('No objects to concatenate')
ValueError: No objects to concatenate
```

**Comment by ChrisGottsacker** [comment]
The first set of stderr messages (i.e. those like std=.0.000000 has illegal value for SNPs[:][3]) also appear in the stderr of epistasis pipeline, but we've just been ignoring them for now

**Comment by ChrisGottsacker** [comment]
We found that the X chromosome was being encoded as 23 instead of 20 when the --make-bed PLINK flag went into effect. This was because the --mouse flag was missing, and the genotype data was assumed to be human.

Also, after making this change (adding the --mouse flag), we ran into another error because mitochondrial genotypes were suddenly included under the 'M' chromosome. To prevent this, the HMDP PLINK view was altered to exclude all mitochondrial markers.

These changes fix the original problem.
